### ISSUE TITLE
tentacle_Component -> Rig building problem

### ISSUE THREAD

**Issue by sbourgoing** [fabric-engine/Kraken#267]
I've found that in the branch develop2.X, it is impossible to correctly generate a tentacle_component from Kraken UI, because the rig generation stop when it is the time to create the input for the tentacleSolverSpliceOp.

I've found a solution: I've removed the "_" character from the following parameter (in tentacle_component.py and TentacleSolver.kl:
*waveLength_Y
*waveAmplitude_Y
*waveFrequency_Y
*waveLength_Z
*waveAmplitude_Z
*waveFrequency_Z

The problem seem to happen because Frabric canvas doesn't allow underscores in the name of it's input/output, so when it is done by script, I guess that the parameter "_" is stripped.

I am currently not setuped correctly to make a pull request, but the change are minor, so maybe it is easier to just make them manually. If you want to do a formal pull request, let me know and I will setup correctly with a fork on my side.

We just started to do R&D on fabric in our studio, so if I found any other problem that I could possibly find a solution, I'll let you know with another issue.

**Comment by EricTRocks** [comment]
Thanks so much for the report Sebastien. develop2.X is our Fabric2.0 beta /
dev branch we are currently working on. This information is very helpful in
our continued work. No need for a pull request. I'll move this bug to the
appropriate milestone in GitHub for us to make sure we cover. It may also
be something we need Fabric Engineers to fix.

--------------------------------------------
Eric Thivierge
http://www.ethivierge.com

**Comment by BigRoy** [comment]
This seems to be "build" under the current develop2.X version (tested in Maya). It builds the guides and builds the rig, though I haven't seen the wave parameters. Probably removed to make it compatible?

**Comment by EricTRocks** [comment]
I think there is a naming issue with '_' characters. We need to take a look
at this issue still.

--------------------------------------------
Eric Thivierge
http://www.ethivierge.com

**Comment by EricTRocks** [comment]
I believe this is a bug that Fabric knows about:
FE-5793 Function Presets - Underscores not usable as port names

**Comment by EricTRocks** [comment]
@sbourgoing This is no longer an issue that I can see when testing on my end with the latest build of Fabric and Kraken. Can you please verify if this is fixed or not on your end please?

**Comment by sbourgoing** [comment]
@EricTRocks Hi Eric ! Right now, i'm not able to confirm that the issue is fixed. I've found this one when I was doing some R&D to integrate fabric in for the rigging pipeline of the compagny i am currently working at. But right now, with all the projects and work to do, I had to stop this R&D (It was a sad moment for my department). As soon as I will be able to get back in Fabric, I will make sure to test if everything is fine on my side.

Thanks a lot for the feedback.
